src/components/NostoProvider.tsx

```diff
@@ -37,8 +37,8 @@ export interface NostoProviderProps {
    * Enables Shopify markets with language and market id
    */
   shopifyMarkets?: {
```

Comment:
For future PR: 
Move this to a type and into nosto-js?

Reply:
From type coupling POV this is a good idea
Downside is that the type def would easily be hidden from Typedoc